specs/unified_kernel_image.md

```diff
@@ -15,12 +15,11 @@ or through a boot loader
 (which is generally useful to allow multiple kernel versions with interactive or
 automatic selection of version to boot into).
 
-Various components of the UKI are provided as PE/COFF sections of the executable.
-The stub is a small program that can be executed in UEFI mode
-that forms the main executable part of the combined image.
-The stub program loads other resources from its file,
-including in particular the kernel and initrd,
-and transitions into the kernel.
+Various components of the UKI are provided as PE/COFF sections of the
+executable.  The stub is a small program that can be executed in UEFI
+mode that forms the initial executable part of the combined image.
+The stub program uses other resources from its executable, including
+in particular the kernel and initrd, and transitions into the kernel.
```

Comment:
I would very much prefer to keep the existing line splits at sentences. This makes diffs much nicer and also makes it easier to review changes.

The previous "the stub program loads" made more sense than the new "the stub program uses". In a sense it is correct, but also misleading, since it just treats them as binary blobs.

Reply:
maybe add the .editoconfig/elisp/vimrc stuff to this repo to make people stick to one line width